addons/im_livechat/models/mail_channel.py

```diff
@@ -87,9 +87,6 @@ def channel_info(self, extra_info=False):
                     channel_infos_dict[channel.id]['operator_pid'] = (res[0], res[1].replace(',', ''))
                 # add the anonymous or partner name
                 channel_infos_dict[channel.id]['livechat_visitor'] = channel._channel_get_livechat_visitor_info()
-                last_msg = self.env['mail.message'].search([("channel_ids", "in", [channel.id])], limit=1)
-                if last_msg:
-                    channel_infos_dict[channel.id]['last_message_id'] = last_msg.id
```

Comment:
This should be moved in `channel_info` in mail (both server and mockserver), so that we always have the last message id available. I think this is the problem, at least for the livechat test.

Reply:
In fact the problem with livechat mock is the way we search for messages in mail mock_server.
In livechat we look for channel_ids in and in mail we look for res_id & model.
So what is the right way to find messages of a channel ?

In `mail`
```js
const messages = this._getRecords('mail.message', [
                ['res_id', '=', channel.id],
                ['model', '=', 'mail.channel'],
                ['message_type', '!=', 'user_notification'],
            ]);
```

Reply:
right way is `channel_ids`
model & id just states the origin thread